### DISCUSSION TITLE
Download model via CLI

### DISCUSSION THREAD

**Discussion by ghost** [openai/whisper#714] [Ideas]
Hello together,

It would be nice to have the option to download the model like [Spacy](https://spacy.io/) with `python -m spacy download $SPACY_MODEL` to avoid using hard-coded URLs in docker file for instance. Then you only have to use `python -m spacy download $WHISPER_MODEL` and don't have to bother about potentially changing URLs or logic to new model versions.

What do you think?

**Comment by nezhar** [comment]
I also needed this and created a PR https://github.com/openai/whisper/pull/789
